package.json

```diff
@@ -5,6 +5,7 @@
     "build": "lerna run build",
     "watch": "lerna run --parallel --stream watch",
     "lint": "lerna run lint",
+    "fmt": "lerna run fmt",
```

Comment:
```suggestion
```
I don't think this belongs to this PR

Reply:
@dwieeb added this in the screen reader plugin and when I created this one I copied some of the same changes. Is this needed? https://github.com/ionic-team/capacitor-plugins/pull/1/files

Reply:
should probably be a separate PR with just that change, he should had probably done that and merged, not a big problem, but only needed once, when your PR or his PR get merged the other will conflict